code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -201,6 +199,7 @@ var/global/list/robot_modules = list(
 				"Sleek - Medical" = "sleekmedic",
 				"Sleek - Chemistry" = "sleekchemistry"
 				)

```

Comment:
Maybe add the unique rescue models here as options? I think that's only `droid-rescue` and `sleekrescue`, but still.